content/docs/components-and-props.md

```diff
@@ -171,11 +170,11 @@ function Avatar(props) {
 }
 ```
 
-The `Avatar` doesn't need to know that it is being rendered inside a `Comment`. This is why we have given its prop a more generic name: `user` rather than `author`.
+इस `Avatar` को जानने की जरूरत नहीं है कि यह एक `Comment` के अंदर रेंडर किया जा रहा है| इसलिए हमने इसके prop को एक ज्यादा सामान्य नाम दिया है : `author` के बजाय `user`|
 
-We recommend naming props from the component's own point of view rather than the context in which it is being used.
+Props का नामकरण हम कौम्पोनॅन्टस के अनुसार करने की सलाह देते हैं बजाय उस संदर्भ में किया जाए जिसमें इसका उपयोग किया जा रहा है|
```

Comment:
can we break this into easier wording instead?

Reply:
Couldn't find simple words. 
I can just reverse the parts ...will that do ??

Reply:
बजाय उस संदर्भ में किया जाए जिसमें इसका उपयोग किया जा रहा है| part is just a bit difficult, could we reword that and split it into 2 sentences that way?